cloud-formation/src/templates/state-machine.yaml

```diff
@@ -26,13 +26,11 @@ States:
   ParallelTasks:
     Type: Parallel
     Branches:
-    - StartAt: ContributionCompleted
+    - StartAt: CheckoutSuccess
       States:
-        ContributionCompleted:
-          Type: Task
-          Resource: "${ContributionCompletedLambda.Arn}"
+        CheckoutSuccess: #Do not rename this step as it is used by support-frontend's polling logic.
+          Type: Pass
```

Comment:
There's no obvious task type which fits what we want to do here - perhaps Pass is the closest available thing: https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-states.html